components/Navbar.tsx

```diff
@@ -0,0 +1,278 @@
+      { title: "Jupyter Hub", href: "/services/classroom#jupyterhub" },
+    ],
+  },
+  {
+    title: "Our Work",
+    href: "/our-work",
+    routes: [
+      { title: "Collaborations", href: "#" },
+      {
+        title: "Software",
+        href: "/our-work/software",
+      },
+      {
+        title: "Workshops and Talks",
+        href: "/our-work/workshops-and-talks",
+      },
+      {
+        title: "Publications",
+        href: "https://publications.ccv.brown.edu",
+      },
+    ],
+  },
+  {
+    title: "Help",
+    href: "/help",
+    routes: [
+      { title: "Submit a Ticket", href: "mailto:support@ccv.brown.edu" },
+      {
+        title: "Discourse",
+        href: "https://ask.cyberinfrastructure.org/c/brown-research-computing/37",
+      },
+      {
+        title: "Slack",
+        href: "https://ccv-share.slack.com/signup#/",
+      },
+      {
+        title: "Office Hours",
+        href: "https://events.brown.edu/ccv/month",
+      },
+    ],
+  },
+  {
+    title: "About",
+    href: "/about",
+    routes: [
+      { title: "Mission", href: "/about#mission" },
+      {
+        title: "Office of Information Technology",
+        href: "/about#office-of-information-technology",
+      },
+      {
+        title: "Our Teams",
+        href: "/about#our-teams",
+      },
+      {
+        title: "People",
+        href: "/about#people",
+      },
+      {
+        title: "Opportunities",
+        href: "/about#opportunities",
+      },
+      {
+        title: "Events",
+        href: "https://events.brown.edu/ccv/month",
+      },
+      {
+        title: "Facilities Statement",
+        href: "/about#facilities-statement",
+      },
+      {
+        title: "Diversity Statement",
+        href: "/about#diversity-statement",
+      },
+    ],
+  },
+  {
+    title: "Blog",
+    href: "/blog",
+    routes: [],
+  },
+]
+
+const Navbar = () => {
+  return (
+    <div className="bg-neutral-50 p-2 shadow flex justify-between items-center">
+      {/* Logo Section */}
+      <section className="flex items-center space-x-3">
+        <a href="https://it.brown.edu">
+          <LogoBrown width={135} />
+        </a>
+        <Link href={"/"}>
+          <LogoCcv width={105} />
+        </Link>
+      </section>
+
+      {/* Navigation Menu for Desktop */}
+      <NavigationMenu.Root className="hidden lg:block">
+        <NavigationMenu.List className="flex list-none space-x-5">
+          {navigation.map((item) => {
+            return (
+              <NavigationMenu.Item key={item.href}>
+                <NavigationMenu.Trigger className="group inline-flex h-9 items-center justify-center rounded-md bg-background px-4 py-2 text-md font-medium transition-colors hover:bg-accent hover:text-accent-foreground focus:bg-accent focus:text-accent-foreground focus:outline-none disabled:pointer-events-none disabled:opacity-50 data-[active]:bg-accent/50 data-[state=open]:bg-accent/50">
```

Comment:
If there's a downside to tailwind, it's that my brain refuses to parse this lol. I'll try and look at it on my machine to give visual feedback, but. that's three lines of classes on my screen 😭.

Reply:
So true!! And using tailwindUI/Shadcn will make this more common